services/net/capture/CaptureAction.cs

```diff
@@ -85,6 +88,25 @@ public CaptureAction(IApiService api, IOptions<CaptureOptions> options, ILogger<
         }
     }
 
+    /// <summary>
+    /// Stop the specified process.
+    /// </summary>
+    /// <param name="process"></param>
+    /// <param name="cancellationToken"></param>
+    /// <returns></returns>
+    protected override async Task StopProcessAsync(ICommandProcess process, CancellationToken cancellationToken = default)
+    {
+        var args = process.Process.StartInfo.Arguments;
+        this.Logger.LogInformation("Stopping process for command '{args}'", args);
+        if (IsRunning(process) && !process.Process.HasExited)
+        {
+            var writer = process.Process.StandardInput;
+            await writer.WriteLineAsync("q");
```

Comment:
To successfully stop `ffmpeg` so that it completes a video file and does not corrupt requires pressing `q`.